src/styles/main.scss

```diff
@@ -1,3 +1,55 @@
 body {
   margin: 0;
 }
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(3, minmax(300px, 1fr));
+
+  text-align: center;
+  line-height: 300px;
+  font-family: inherit;
+  font-size: 100px;
+  color: white;
+
+  &__block--1 {
```

Comment:
It's better to use collection with @each loop here, like we did in calendar task